tox.ini

```diff
@@ -11,20 +11,18 @@ usedevelop = false
 
 [testenv]
 description =
-  Run the tests under {basepython} and
```

Comment:
Why is "and" unnecessary? isn't it merged with the following lines?

Reply:
Run `tox -va` locally and you will understand. Before making this change it was producing wrong text due to sentence ending on "and" for the simple python envs.

So far that approach was the only one that produced relatively good descriptions:
![](https://sbarnea.com/ss/Screen-Shot-2021-06-24-15-21-34.20.png)